docs/docs/self-hosting/manage/updating_scaling.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Updating and Scaling
+---
+
+This page shows, how you can improve your day-two operations
+by separating the init and setup phases from the ZITADEL start runtime process.
+
+## Scaling ZITADEL
+
+For production setups, we recommend that you run  ZITADEL in a highly available mode.
+As all shared state is managed in the database,
+the ZITADEL binary itself is stateless.
+You can easily run as many ZITADEL binaries in parallel as you want.
+If you use [Knative](/docs/self-hosting/deploy/knative)
+or [Google Cloud Run](https://cloud.google.com/run) (which uses Knative, too),
+you can even scale to zero.
+Especially if you use an autoscaler that scales to zero,
+it is crucial that you minimise startup times.
+
+## Updating ZITADEL
+
+When updating ZITADEL, the setup job takes care of database migrations.
+You generally want to run the job in a controlled manner,
+so multiple executions don’t interfere with each other.
+Also, after the setup is done,
+rolling out a new ZITADEL version is much faster
+when the runtime processes are just executed with zitadel start.
+
+## Separating Init and Setup from the Runtime
+
+If you use the [official ZITADEL Helm chart](/docs/self-hosting/deploy/kubernetes),
+then you can stop reading now,
+as the init and setup phases are already separated and executed in dedicated Kubernetes Jobs.
+If you use another orchestrator and want to separate the phases yourself,
+you should know what happens during these phases.
+
+### The Init Phase
+
+The command `zitadel init` ensures the database connection is ready to use for the subsequent phases.
+It just needs to be executed once over ZITADELs full life cycle,
+when you install ZITADEL from scratch.
+Using the configured database admin user, `zitadel init` ensures the following:
```

Comment:
While reading this I got confused by the mention of a specific user. How can I define or run `zitadel init` with a  different user? Do I need to add some parameters to execute as admin user?
Can we link this here or remove the reference to the admin user?

Reply:
The admin user is important and has to exist already. Is that more clear with the latest commit?